pkg/kubelet/apis/config/types.go

```diff
@@ -353,12 +353,13 @@ type KubeletConfiguration struct {
 
 	// A set of ResourceName=ResourceQuantity (e.g. cpu=200m,memory=150G,pid=100) pairs
 	// that describe resources reserved for non-kubernetes components.
-	// Currently only cpu and memory are supported.
+	// Currently only cpu, memory, local ephemeral storage for root file system, pid (process IDs) and swap are supported
+	// Since 1.22, swap is supported when NodeSwap is enabled
```

Comment:
```suggestion
	// Since 1.24, swap is supported when NodeSwap is enabled
```

Since this wasn't supported in 1.22/1.23